app/views/staff/shared/activities/tree_view/_table_tabbed.html.haml

```diff
@@ -4,9 +4,9 @@
       .govuk-tabs{ data: { module: "govuk-tabs" } }
         %h2.govuk-tabs__title
           Current activities
-        %ul.govuk-tabs__list
+        %ul.govuk-tabs__list{role: "tablist"}
```

Comment:
This addition seems redundant, because this tab structure is getting its markup thanks to the `module: "govuk-tabs"`.
![Screenshot 2021-08-12 at 15 47 27](https://user-images.githubusercontent.com/579522/129218976-cb641fbb-2ba7-4513-b9ea-297b1bc8bde8.png)